shell/platform/windows/flutter_window.h

```diff
@@ -316,6 +316,25 @@ class FlutterWindow : public KeyboardManager::WindowDelegate,
   std::unique_ptr<DirectManipulationOwner> direct_manipulation_owner_;
 
  private:
+  // WM_DPICHANGED_BEFOREPARENT defined in more recent Windows
+  // SDK
+  const static long kWmDpiChangedBeforeParent = 0x02E2;
+
+  // Timer identifier for DirectManipulation gesture polling.
+  const static int kDirectManipulationTimer = 1;
```

Comment:
Here and above, while you're in here, let's also switch these to `static const` for consistency with elsewhere.